app/producers/payload_tracker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# A Kafka producer client for payload-tracker
+# https://www.rubydoc.info/gems/ruby-kafka/Kafka/Producer
+class PayloadTracker < Kafka::Client
+  BROKERS = [ENV['KAFKAMQ']].compact.freeze
+  CLIENT_ID = 'compliance-payload-tracker-producer'
+  TOPIC = 'platform.payload-status'
```

Comment:
I don't know if this is the payload that's actually in the cluster - double check with QE for this. The repo says that https://github.com/RedHatInsights/payload-tracker - but I had to change this to `payload_tracker` to make it work

Reply:
Checked on openshift and this TOPIC is correct - nothing to change here :smile: